apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/LoadedFilterManager.java

```diff
@@ -27,6 +31,10 @@
 public class LoadedFilterManager {
   private static final Logger LOGGER = LoggerFactory.getLogger(LoadedFilterManager.class);
 
+  // A date before the lastUpdate feature was rolled out
+  private static final Date BEFORE_LAST_UPDATE = Date.from(Instant.parse("2020-01-01T00:00:00Z"));
+
+  // The size of the beneficiaryId column
```

Comment:
This date is used in the case that the `LoadedBatches` table is empty. It just needs to be sometime in the past.